### DISCUSSION TITLE
Account Deletion

### DISCUSSION THREAD

**Discussion by Takeda0x64** [codemagic-ci-cd/codemagic-docs#1479] [Q&A] [answered]
How i can delete my code magic account? i could not find a delete button 
email: alan1010035@gmail.com

**Comment by dtrdic** [comment, marked as answer]
Hey @Kenn0x64, the delete account option can be found on this location;
in the sidebar click on **Teams** -> **Personal Account** -> and at the bottom of the account settings you have a **Danger zone** section where you have an option to delete your account

**Reply by Takeda0x64** [reply]
Thanks!
